tests/sentry/ingest/ingest_consumer/test_dlq.py

```diff
@@ -87,3 +90,56 @@ def test_dlq_invalid_messages(factories, topic_name, consumer_type) -> None:
 
         assert exc_info.value.partition == partition
         assert exc_info.value.offset == offset
+
+
+@pytest.mark.parametrize(
+    "topic_name",
+    [
+        TopicNames.INGEST_TRANSACTIONS.value,
+    ],
+)
+@django_db_all
+def test_dlq_stale_messages(factories, topic_name) -> None:
```

Comment:
this test doesn't belong here since this is the ingest consumer mode. We are testing a strategy that is agnostic of any specific consumer. test file structure matches the module being tested so it should be under `consumers/test_dlq`